src/modules/dashboard/application/use-cases/widgets/__tests__/get-activity-feed.use-case.spec.ts

```
@@ -0,0 +1,308 @@
+
+    it('should handle repository errors', async () => {
+      const query: WidgetDataQueryDto = {};
+      const error = new Error('Database error');
+      historyRepository.paginate.mockRejectedValue(error);
+
+      await expect(useCase.execute(query)).rejects.toThrow(error);
+    });
+
+    it('should handle all entity types correctly', async () => {
+      const entities = ['server', 'vm', 'user', 'group', 'room'];
+      const events = entities.map((entity, index) => ({
```

ESLint

[error] Replace `(entity,·index)·=>` with `⏎········(entity,·index)·=>⏎·········`

(prettier/prettier)